src/main/resources/application.yml

```diff
@@ -63,10 +63,11 @@ dateCalculations:
 notifications:
   govNotifyApiKey: ${GOV_NOTIFY_API_KEY:}
   frontendBaseUrl: "${FRONTEND_BASE_URL:}"
+  respondToClaimUrl: "${RESPOND_TO_CLAIM_URL:}"
   templates:
     email:
       claimantClaimIssued: "58e645b8-bcd6-4462-822c-f17da9dd3dc7"
-      defendantClaimIssued: "c38550a0-a981-4698-a644-481ad3dd8228"
+      defendantClaimIssued: "49027a97-412e-45ee-86df-bf2fa6df6d8c"
```

Comment:
I would have thought we would use the existing one and just update the parameter on the day?

Reply:
Had that idea as well but wanted have deployment that does not require human intervention. 

With web hooks in place we would need to be doing it this anyway.

Other thing is that in the future I would like to have all URLs configurable via application config, no updates on GOV.UK Notify, mainly to have single place to change.